### PR TITLE
[ADD] contract_commission

### PR DESCRIPTION
**Pull request by pedrobaeza**
# 
# Commission in contracts

This module allows to get default commissions for partner in the recurring
invoices generated from contracts.
# Installation

This module is auto-installed when you have _account_analytic_analysis_ and
_sale_commission_ modules installed.
# Usage

When invoicing contracts, the agents assigned to the partner of the contract
will now be assigned to the generated invoices.

### CODE REVIEW

File: contract_commission/README.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+============
+
+This module is auto-installed when you have *account_analytic_analysis* and
+*sale_commission* modules installed.
+
+Usage
+=====
+
+When invoicing contracts, now agents assigned to the partner of the contract
+will be assigned on the generated invoices
```
Comment (elicoidal) [inline contract_commission/README.rst]: When invoicing contracts, the agents assigned to the partner of the contract
will now be assigned to the generated invoices

Comment (pedrobaeza) [inline contract_commission/README.rst]: Thanks for using your translations skills in reviewing my PRs! Change done.

**Comment by angelmoya** [comment]
Tested on runbot OK, :+1: 

It would be usefull allow to set agent in contract, because it's possible that set diferent commisions on contract and other sales, but this works fine.

File: contract_commission/README.rst (hunk 1 above)
Comment (elicoidal) [inline contract_commission/README.rst]: bah: given my limited technical knowledge I try to help the best I can!

**Comment by pedrobaeza** [comment]
@amoya-dx, I have added your suggestion in Known issues / Roadmap section.

File: contract_commission/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': 'Commissions in contract invoices',
+    'version': '8.0.1.0.0',
+    'author': 'Serv. Tecnol. Avanzados - Pedro M. Baeza',
```
Comment (Cedric-Pigeon) [inline contract_commission/__openerp__.py]: Could you please add Odoo Community Association (OCA) in author section ?

**Comment by Cedric-Pigeon** [comment]
Small remark otherwise :+1:  Thanks !

**Comment by pedrobaeza** [comment]
@Cedric-Pigeon, thanks for the review. It's fixed and you can proceed with the merge.

**Comment by Cedric-Pigeon** [comment]
@pedrobaeza Thanks for the fix but I haven't got any merge rights ;-)

**Comment by pedrobaeza** [comment]
@elicoidal, can you proceed with the merge?

File: contract_commission/README.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+* https://openclipart.org/detail/43969/pile-of-golden-coins-by-j_alves
+* Subicon made by `Freepik <http://www.flaticon.com/authors/freepik>_ from
+  www.flaticon.com
+
+Maintainer
+----------
+
+.. image:: http://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
```
Comment (elicoidal) [inline contract_commission/README.rst]: https

File: contract_commission/README.rst [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline contract_commission/README.rst]: https

File: contract_commission/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    "category": "Generic Modules/Sales & Purchases",
+    'license': 'AGPL-3',
+    'depends': [
+        'account_analytic_analysis',
+        'sale_commission',
+    ],
+    'contributors': [
+        "Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>",
+    ],
+    "data": [],
```
Comment (elicoidal) [inline contract_commission/__openerp__.py]: Remove empty keys

File: contract_commission/models/account_analytic_account.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, api
```
Comment (elicoidal) [inline contract_commission/models/account_analytic_account.py]: Please sort in alphabetical order following: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

**Comment by elicoidal** [comment]
small details. Once corrected I will merge it

**Comment by pedrobaeza** [comment]
@elicoidal, done!

**Comment by elicoidal** [comment]
Merged!

**Comment by pedrobaeza** [comment]
Thanks!

**Pull request merged**